test-fixtures/checksum-file/sha512-bsd.sum

```diff
@@ -0,0 +1 @@
+SHA512 (content.txt) = 060a8cc41c501e41b4537029661090597aeb4366702ac3cae8959f24b2c49005d6bd339833ebbeb481b127ac822d70b937c1637c8d0eaf81b6979d4c1d75d0e1
```

Comment:
Just curious, do we have examples of this in the wild? I can find citations for MD5 but not necessarily SHA.

I don't think we need to worry about blocking these cases if we don't find any though as the implementation is simpler by accounting for these as a result (as the algo is lowercased generally and exceptions would be of low value).

Reply:
I could not find examples of this in the wild no so I made them up ! Okay removing them 🙂

Reply:
I was more specifically referring to the `parseChecksumLine` function's behavior of reading hash type from the file, but again, don't change anything - it's fine as is.

Reply:
you mean we could just go ahead and just guess the checksum from it's len ?